lib/features/children/add_member/cubit/add_member_cubit.dart

```diff
@@ -49,6 +49,16 @@ class AddMemberCubit extends Cubit<AddMemberState> {
   }
 
   void allFormsFilled() {
+    if (state.members.where((element) => element.type == 'Child').isEmpty) {
```

Comment:
another good candidate for getters proposed in another ;)

Reply:
also I'm a bit confused that now we have both Profile and Member models in the app.. Could you please provide more historical context here?

Reply:
Member is the model we need to add a member to the family group: children or adults. Profile is what we get returned from BE including monster picture & wallet amount.